frontend/src/lib/components/icons.tsx

```diff
@@ -1,7 +1,21 @@
 // Loads custom icons (some icons may come from a third-party library)
-import React, { CSSProperties } from 'react'
+import React, { CSSProperties, PropsWithChildren } from 'react'
 import './icons.scss'
 
+interface IconWithCountProps {
+    count: number
+    showZero?: boolean
+}
+
+export function IconWithCount({ count, showZero, children }: PropsWithChildren<IconWithCountProps>): JSX.Element {
+    return (
+        <div style={{ position: 'relative' }}>
```

Comment:
The sizing of this seems a bit off: it's 24x25px in "Add to dashboard" and in Storybook it looks like weirdly like this:

<img width="47" alt="Screen Shot 2022-04-28 at 13 59 51" src="https://user-images.githubusercontent.com/4550621/165747698-e9477bc7-232b-4bfc-9fef-35ea80a9340c.png">

`IconWithCount` should be exactly the same size as the underlying icon (so 24x24px by default, or 1em by 1em).